nltk/translate/api.py

```diff
@@ -275,19 +274,33 @@ class Alignment(frozenset):
         >>> b = Alignment([(0, 0), (0, 1)])
         >>> b.issubset(a)
         True
-        >>> c = Alignment('0-0 0-1')
+        >>> c = Alignment.fromstring('0-0 0-1')
         >>> b == c
         True
     """
 
-    def __new__(cls, string_or_pairs):
-        if isinstance(string_or_pairs, string_types):
```

Comment:
Since `string_types` is no longer used, the import name in https://github.com/stevenbird/nltk/blob/ae2d80f24b6cdf739bdca7ef32584e9f5839abd2/nltk/translate/api.py#L12 can be removed.